pkg/gke/update.go

```diff
@@ -518,7 +519,7 @@ func UpdateNodePoolManagement(
 		needsUpdate = true
 	}
 	if needsUpdate {
-		logrus.Infof("updating management config of node pool [%s] on cluster [%s]", *nodePool.Name, config.Name)
+		logrus.Infof("updating management config (upstream: %+v, config: %+v) of node pool [%s] on cluster [%s (%s)]", upstreamNodePool.Management, nodePool.Management, *nodePool.Name, config.Spec.ClusterName, config.Name)
```

Comment:
it is wrong, NodePool names can't be different from upstream. Using `%+v` as formatting verb is really bad idea.